lambda_handlers/tests/unit/test_hubapi_list.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import json
+import logging
+import pymongo
+import pytest
+import mongomock
+import mock
+
+from jina.docker.hubio import HubIO
+
+from ...hubapi_list import MongoDBHandler
+from ...hubapi_list import lambda_handler
+from jina.helper import yaml
+
+
+def get_logger(context='generic', file=True):
+    logger = logging.getLogger(context)
+    logger.setLevel(logging.DEBUG)
+    return logger
+
+@mock.patch("lambda_handlers.hubapi_list.MongoDBHandler.connect")
```

Comment:
I think it is better if possible to use `monkeypatch` right? doesn't it collide with the `@mock.patch`?

Reply:
Since we use `MongoDBHandler` as a context manager, it is better to mock the return values of the context managers & test